components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -59,7 +60,7 @@ class GeckoEngine(
     executorProvider: () -> GeckoWebExecutor = { GeckoWebExecutor(runtime) },
     override val trackingProtectionExceptionStore: TrackingProtectionExceptionStorage =
         TrackingProtectionExceptionFileStorage(context, runtime)
-) : Engine {
+) : Engine, WebExtensionManager {
```

Comment:
We talked about the name some more and think `WebExtensionRuntime` is better than introduce yet another "manager" class. Do you agree? If so, then we update this one more time before landing. Thanks!

Reply:
Works for me, choosing names is always hard :)